wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/ui/component/EpisodeListScreen.kt

```diff
@@ -0,0 +1,121 @@
+package au.com.shiftyjelly.pocketcasts.wear.ui.component
+
+import androidx.annotation.StringRes
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.wear.compose.foundation.lazy.items
+import androidx.wear.tooling.preview.devices.WearDevices
+import au.com.shiftyjelly.pocketcasts.models.entity.PodcastEpisode
+import au.com.shiftyjelly.pocketcasts.wear.theme.WearAppTheme
+import com.google.android.horologist.compose.layout.ScalingLazyColumn
+import com.google.android.horologist.compose.layout.ScalingLazyColumnState
+import java.util.Date
+import au.com.shiftyjelly.pocketcasts.localization.R as LR
+
+@Composable
+fun EpisodeListScreen(
+    columnState: ScalingLazyColumnState,
+    uiState: EpisodeListUiState,
+    @StringRes title: Int,
+    useEpisodeArtwork: Boolean,
+    onItemClick: (PodcastEpisode) -> Unit,
+) {
```

Comment:
This extracts the episode list logic from the Downloads screen so it could be reused.